src/main/java/Deadline.java

```diff
@@ -0,0 +1,18 @@
+public class Deadline extends Task {
+    protected String by;
+
+    public Deadline(String description, String by) {
+        super(description);
+        this.by = by;
+    }
+
+    @Override
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : " ");
```

Comment:
Would it be a good idea to add a comment to explain the unicode so that people know what you are returning?

Reply:
yes, definitely, will be adding comments, thank you!